packages/infra/lib/hl7-notification-stack/mllp.ts

```diff
@@ -58,12 +59,23 @@ const setupNlb = (identifier: string, vpc: ec2.Vpc, nlb: elbv2.NetworkLoadBalanc
   return targetGroup;
 };
 
+const createHieTimezoneMap = (hieConfigs: Record<string, HieConfig>) => {
+  return Object.values(hieConfigs).reduce((acc, item) => {
+    const key = item.identifierInMshSegment;
+    const val = item.timezone;
+    if (key && val) {
+      acc[key] = val;
+    }
+    return acc;
+  }, {} as Record<string, string>);
+};
```

Comment:
@coderabbitai you should've flagged this as we avoid using const/arrow functions, as indicated to you before a few times.